packages/api-explorer/src/StandaloneApiExplorer.tsx

```diff
@@ -86,15 +85,9 @@ export const StandaloneApiExplorer: FC<StandaloneApiExplorerProps> = ({
     }
   }, [versionsUrl, currentVersionsUrl])
 
-  const chosenSdk: IAPIMethods = initRunItSdk(defaultConfigurator)
-
   return (
     <Provider store={store}>
-      <RunItProvider
-        sdk={chosenSdk}
-        configurator={defaultConfigurator}
-        basePath="/api/4.0"
-      >
+      <RunItProvider configurator={defaultConfigurator} basePath="/api/4.0">
```

Comment:
This is good for now but as discussed we're going to need to abstract that "basePath" value, and come up with a story/UX for stand-alone Looker API 3.1 calls ... or we just let the error handler fire for 3.1 requests that violate CORS

Reply:
Agreed